business/istio_config.go

```diff
@@ -458,6 +458,8 @@ func getUpdateDeletePermissions(k8s kubernetes.IstioClientInterface, namespace,
 			for _, ssar := range ssars {
 				if ssar.Spec.ResourceAttributes != nil {
 					switch ssar.Spec.ResourceAttributes.Verb {
+					case "create":
```

Comment:
line 456 is missing the "create" argument to k8s.GetSelfSubjectAccessReview.

Reply:
ups, true, good catch !